### DISCUSSION TITLE
Exposing an HTTPS endpoint

### DISCUSSION THREAD

**Discussion by rajataghi** [gotenberg/gotenberg#502] [Q&A] [answered]
I have the gotenberg image hosted in an ECS container on aws and I'm able to send http requests to the container on port 3000 to convert a word file to pdf on the libreoffice endpoint.
However, that is an HTTP endpoint (just like in the curl request examples on the website), is it possible to expose an HTTPS endpoint for the docx -> pdf conversion?

Thanks

**Comment by gulien** [comment, marked as answer]
Hello @rajataghi,

Gotenberg does not expose an HTTPS endpoint because it’s not its role. You should use a proxy that handles the HTTPS part for you.
